packages/rocketchat-cors/cors.js

```diff
@@ -0,0 +1,107 @@
+/* globals WebAppInternals */
+
+import url from 'url';
+
+WebApp.rawConnectHandlers.use(function(req, res, next) {
+	if (req._body) {
+		return next();
+	}
+	if (req.headers['transfer-encoding'] === void 0 && isNaN(req.headers['content-length'])) {
```

Comment:
Change this `void 0`, as the majority of people aren't going to know what this means 100%.